run_scalefactor_creations.sh

```diff
@@ -1,23 +1,46 @@
-eras=("2016preVFPUL" "2016postVFPUL")
+eras=("2017UL" "2018UL" "2016preVFPUL" "2016postVFPUL")
 channels=("muon" "embeddingselection")
 
-output_dir=output
+output_dir="output"
 
 mkdir -p $output_dir
-cp -r settings $output_dir/used_settings # Copy settings that are used to output directory
+cp -r settings $output_dir/used_settings         # Copy settings that are used to output directory
+cp set_inputfiles.yaml $output_dir/used_settings # Copy input file list to output directory
+
+# ---
 
 for era_idx in $(seq 0 $((${#eras[@]} - 1))); do
 	(
 		era=${eras[$era_idx]}
 		for channel_idx in $(seq 0 $((${#channels[@]} - 1))); do
 			(
 				channel=${channels[$channel_idx]}
-				python3 scripts/TagAndProbe.py --channel $channel --era $era
-				if [[ "$channel" == "embeddingselection" ]] && ([[ "$era" == "2016preVFPUL" ]] || [[ "$era" == "2016postVFPUL" ]]); then # Additional dz quantity of trg_Mu17TrkMu8_DZ_Mu17
-					python3 scripts/TagAndProbe.py --channel $channel --era $era --no-leg-switching --mode="UPDATE"
+				nice -n 19 python3 scripts/TagAndProbe.py \
```

Comment:
nice -n 19 not needed imo

Reply:
I would suggest keeping it. Depending on the combination of channels and eras, this process can occupy 8+ cores for (quite) some time. Note that in case of i.e. the muon channel Data, MC and Embedding is processed in parallel adding up to the 8 mentioned cores.